tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
OpenThread configures the minimum DTLS handshake timeout to 8 seconds.  I'm guessing that a DTLS handshake message is getting lost somewhere.  Can you enable some more logs to show the DTLS messages being sent/received?